### ISSUE TITLE
nano-theme does not set the background in terminal mode in the first frame

### ISSUE THREAD

**Issue by ronisbr** [rougier/nano-theme#47]
Hi!

I am trying to use nano-theme in the terminal mode. It seems that it does not set the background when the theme is loaded. Here is the screenshot just after `load-theme`:

<img width="762" alt="Captura de Tela 2023-02-03 às 09 41 21" src="https://user-images.githubusercontent.com/1068295/216606119-b058ee8d-50a1-4692-bc83-382b6fb698da.png">

However, if I create a new frame with `nano-new-frame`, then the background is set to the correct color:

<img width="718" alt="Captura de Tela 2023-02-03 às 09 41 52" src="https://user-images.githubusercontent.com/1068295/216606218-6c09d299-524a-4f18-b050-a856d7c8081a.png">

**Comment by rougier** [comment]
That' surprising that `nano-new-frame` is even working since it is supposed to work only with a gui. Do the new frame overwrite the current window ? (I guess you don't get a new terminal window)

**Comment by ronisbr** [comment]
Yes! It overwrite the current window. It seems to open "another" Emacs, since I need to exit twice to quit Emacs.

**Comment by rougier** [comment]
Interesting. `nano-new-frame` should then check for GUI before trying to create an new frame. Concerning the background color, I'm not quite sure what is happening Could you give a try to nano-theme (on ELPA) and check if it has the same problem?
